api/desecapi/urls/version_1.py

```diff
@@ -18,6 +21,7 @@
 
     # Token management
     path('tokens/', include(tokens_router.urls)),
+    path('tokens/<token_id>/domain_policies/', include(tokendomainpolicies_router.urls)),
```

Comment:
I don't know. Perhaps role policies, at some point?

Reply:
`tokens/<token_id>/policies/domain`?

Reply:
Why is that better?

I am not opposed, and I like the idea semantically because it is more extensible. But, it also kind of implies that something non-404 will happen if a request is sent to `policies/`. What would that be?

Reminds me of DNS empty non-terminals ...

Reply:
as discussed offline, added "overview" endpoint similar to api root